### DISCUSSION TITLE
依依的城市 | /posts/jingmen/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#649] [Announcements]
https://yuanfan.rbind.io/posts/jingmen/

**Comment by meeeeeeeeo** [comment]
> 正好赶上高铁开通

是荆门吗  😂

---
注意到了blog的名字jingmen，看来是的  😂

**Reply by earfanfan** [reply]
是的呀，你去过吗？那是个好地方，托依依的福，我吃到了好多好吃的。

**Reply by meeeeeeeeo** [reply]
还没去过，我女朋友钟祥的，有些亲戚在荆门，一直有计划去下。
以前回钟祥/荆门要坐三个多小时的绿皮火车，我们盼高铁也盼了好久，只等明年高铁通到钟祥

**Reply by earfanfan** [reply]
其实我真的是赶上了，我去是上周五下班了去，坐顺风车过去也是3小时，然后到了周六逛公园的时候，堂妹依依告诉我高铁开通了，于是坐了2小时高铁就回武汉了。话说荆门西站真的修得非常高大上，特别亮堂，空间很大，电子化程度更高。我跟依依还吐槽说，这就是后发优势。

**Comment by tianheg** [comment]
8幅画都很好看，我猜第1张图片右下角那个是你画的，觉得花朵的形状有些“粗放、随意”，我猜对了吗？

**Reply by earfanfan** [reply]
博客里面的8张都是依依画的。不过啊，终于有人夸画好看了，我先给你分享一下我们最近画的吧。

下面这是依依最近画的，第一张猫猫云很可爱吧。
![20241227-01](https://github.com/user-attachments/assets/3f1cca9f-a88a-42a0-9595-d30060363cc0)
![20241227-02](https://github.com/user-attachments/assets/71677f9d-bd8e-48da-8c88-e536b02a3c0b)
![20241227-03](https://github.com/user-attachments/assets/1414aa11-434a-450d-8fe9-3da23e865326)
![20241227-04](https://github.com/user-attachments/assets/44631f92-2677-4477-8d36-5909c5d21c50)
![20241227-05](https://github.com/user-attachments/assets/58b9f086-4245-40c6-8105-3d8ea7486bdd)

和依依的画相比，我画的都比较简单，下面是我最近画的，请看。
![20241227-06](https://github.com/user-attachments/assets/554e8837-eb21-456b-9fc4-23e0d1401579)

之前给好朋友看让她猜的是下面这两张画，月亮在海面的倒影画成了海浪……不用我说，你一定能猜对哪个是我画的。
![20241227-07](https://github.com/user-attachments/assets/505f22eb-0bd9-4673-be4b-8a16986eb99f)

如果我没记错的话，你的头像是不是又改回来了？之前好像改成衣服洗桶那个希腊字母来着。

**Reply by tianheg** [reply]
猫猫云确实很可爱，画的绿色草地一眼望去，给我一种生机勃勃的感觉。



小王子和他的玫瑰，这本书想看很久了，总是因为其他事情而搁置。



你画得更简单，可能也更具想象空间？能给人更多的思考。



关于我的头像：



是滴！我改回来了。还是觉得自己设计的这个头像更能代表自己。咱们上次讨论我的头像是在这篇评论里，我给翻出来了 https://github.com/earfanfan/yuanfan/discussions/606#discussioncomment-8655072

**Reply by earfanfan** [reply]
>小王子和他的玫瑰，这本书想看很久了，总是因为其他事情而搁置。

我也有很多想看的书，想看的电影电视剧，还有好多想做的事，都还在排队中。不过也有很多事排队的时间长了，就从我脑子里消失了。

其实我画的就是偏向简单，大多就是色块平涂，也没撒想象空间，哈哈，因为我还是很新的新手。下面的画都是我画的，不过只有一张是我自己想的，算是有一点点想象力，其他都是参考网上搜集的教程按步骤一步步来的，你不妨猜猜是哪张？

![20241229](https://github.com/user-attachments/assets/f23bbaac-ece0-46d0-9734-ed2105f7da74)

其实我总觉得，好像你以前也在哪里说起过想看小王子很久，但一直没看。

**Reply by tianheg** [reply]
我猜是第3行第2个

> 其实我总觉得，好像你以前也在哪里说起过想看小王子很久，但一直没看。

haha，这个坑今年一定补上。25年年度主题之一就是多读书

**Reply by earfanfan** [reply]
>我猜是第3行第2个

是第3行从右往左数第2个，一个人坐在月亮上钓鲸鱼。
